charts/internal/cloud-provider-config/templates/auditsink-config.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: v1
```

Comment:
Why not just deploy the `AuditSink` resource directly together with the webhook deployment from the control-plane charts? I would prefer this to be encapsulated by the "enabled" condition as well.

Reply:
Right - will do this.